data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,558 @@ mission "Heliarchs Buy Jump Drive Mission"
+
+
+mission "Coalition: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Anna Fangvin has completed the registration process, help her get a translation device on <destination>."
+	source "Ring of Friendship"
+	destination "Ki Patek Ka"
+	to offer
+		has "Coalition: Struggling Author 2: done"
+	on offer
+		conversation
+			`Going through the spaceport looking for Anna, you finally find her at a restaurant serving food made for Saryds. She's typing away furiously at her laptop, with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
+			`	You sit down next to Anna. "These vegetables are so delicious! After months of instant noodles and takeout back home, it finally feels good to eat something that's actually healthy." she exclaims, closing the computer.`
+			choice
+				`	"How good is it?"`
+					goto good
+				`	"What did you usually eat back in human space?"`
+			`	"My diet wasn't exactly the best. All I had was carbs and takeout food; I never could find time in my schedule to cook at home. I was working hard on 'Beyond the Hyperdrive' at the time. This food is better than anything I could ever find on Zug.`
+				goto published
+			label good
+			`	"It's very good. Probably better than anything I've had back on Zug. Now, I'd like to get my book published here."`
+				goto company
+			label published
+			`	"On another note, I want to get my book published here."`
+			label company
+			`	"Have you found a publishing company yet?" you ask.`
+			`	"No," Anna replies, then begins complaining about how it was extremely difficult for her to communicate with the restaurant staff, who did not have a translation box.`
+			choice
+				`	"I can get you a translator. It shouldn't cost anything."`
+				`	"I'd be happy to help obtain a translator. I'm guessing we can get one for free."`
+			`	"Great! So, where will we get the translator?"`
+			`	"We'll go to <destination> to ask the Heliarchs for a translation box," you tell Anna. "Finish your meal, then go back to my ship. I'll be ready." You walk off to your ship.`
+			`	After a few minutes, you see Anna walking towards you. You open the hatch as she enters. She settles into her bunk as you prepare to depart.`
+				accept
+	on complete
+		conversation
+			`You arrive at the spaceport. Anna leaves her bunk and waits for you to open the hatch. You quickly lead her to a Heliarch office. You walk to the counter, and begin conversing with the clerk.`
+			`	"Greetings, human. To be of assistance to you, how can I?" the clerk says.`
+			`	Anna speaks up. "I'm Anna Fangvin, and I'd like to publish a book here in Coalition space. My friend <first> <last> tells me I will need a translation box to converse with the publishing company. Could you provide one?"`
```

Comment:
The publishing company would have a translator, I'd suggest altering the focus of this to being about Anna looking up different publishers and finding one that seemed more promising for her books' stories.

There can still be the Heliarch assisting you and Anna, would be kinda like them keeping an eye on it all, it's just the focus on getting the translator right away for her that I'd shift, as the player should know it's not that simple to get one.